app/assets/stylesheets/components/_slabs.scss

```diff
@@ -24,6 +24,10 @@
   }
 }
 
+.slab--stimulus-cta {
+  background-color: $color-stimulus-blue;
+}
+
```

Comment:
Non-blocking, but for CSS cleanliness, it looks like we could be reusing css classes more effectively here, for example, the slab could use more classes as if they were "mixins", while keeping these css class definition more modular.
```html
<!-- this allows us to keep only one css definition per class -->
<div class="slab slab--cta slab--stimulus">
</div>
```

Reply:
cool i'll change this. i chose not to do this initially because there is exactly one other usage of slab--cta so it seemed like a toss-up.

Reply:
done!